SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -223,9 +226,98 @@ void USpatialWorkerConnection::ConnectToLocator()
 	});
 }
 
-bool USpatialWorkerConnection::ShouldConnectWithLocator()
+void USpatialWorkerConnection::ConnectToLocator()
+{
+	if (LocatorConfig.WorkerType.IsEmpty())
+	{
+		LocatorConfig.WorkerType = SpatialConstants::ClientWorkerType;
+	}
+
+	if (LocatorConfig.WorkerId.IsEmpty())
+	{
+		LocatorConfig.WorkerId = LocatorConfig.WorkerType + FGuid::NewGuid().ToString();
+	}
+
+	FTCHARToUTF8 PlayerIdentityTokenCStr(*LocatorConfig.PlayerIdentityToken);
+	FTCHARToUTF8 LoginTokenCStr(*LocatorConfig.LoginToken);
+
+	Worker_Alpha_LocatorParameters LocatorParams = {};
+	LocatorParams.player_identity.player_identity_token = PlayerIdentityTokenCStr.Get();
+	LocatorParams.player_identity.login_token = LoginTokenCStr.Get();
+
+	WorkerLocator = Worker_Alpha_Locator_Create(TCHAR_TO_UTF8(*LocatorConfig.LocatorHost), 0, &LocatorParams);
```

Comment:
I believe we want to call `Worker_Alpha_Locator_Destroy` in `DestroyConnection` if `WorkerLocator != nullptr`.

Reply:
We are now calling `Worker_Alpha_Locator_Destroy` in `DestroyConnection`